schema.graphql

```diff
@@ -167,64 +163,52 @@ type Withdrawal implements Action @entity {
   account: Account!
   amount: BigInt!
   shares: BigInt!
-  pricePerFullShare: BigInt!
-  vaultBalance: BigInt!
-  totalSupply: BigInt!
-  available: BigInt!
+  # TODO: check best way to incorporate this updates
+  vaultStats: VaultUpdate!
   transaction: Transaction!
 }
 
 type Harvest @entity {
   id: ID!
   vault: Vault!
   strategy: Strategy!
-  caller: Bytes!
-  pricePerFullShareBefore: BigDecimal!
-  pricePerFullShareAfter: BigDecimal!
-  vaultBalanceBefore: BigDecimal!
-  vaultBalanceAfter: BigDecimal!
-  strategyBalanceBefore: BigDecimal!
-  strategyBalanceAfter: BigDecimal!
-  earnings: BigDecimal!
-  pricePerFullShareBeforeRaw: BigInt!
-  pricePerFullShareAfterRaw: BigInt!
-  vaultBalanceBeforeRaw: BigInt!
-  vaultBalanceAfterRaw: BigInt!
-  strategyBalanceBeforeRaw: BigInt!
-  strategyBalanceAfterRaw: BigInt!
-  earningsRaw: BigInt!
+  harvester: Bytes!
+
+  vaultUpdate: VaultUpdate!
+  strategyUpdate: StrategyUpdate!
+
   transaction: Transaction!
 }
 
 type Strategy @entity {
```

Comment:
Add "name" to strategy, as well as strategy metadata.. active/archived/winding/unwinding